### ISSUE TITLE
[BUG] (Single line quick fix)

### ISSUE THREAD

**Issue by ahiser1117** [genjax-community/genjax#621]
I noticed that on line 45 of src/genjax/_src/inference/smc/rejuvenate.py you currently have
```
sub_keys = jax.random.split(key)
```
but I believe it should be.
```
sub_keys = jax.random.split(key, n_particles)
```
so that the shapes in the vmap function on line 46 are the same between the first two parameters.

**Comment by femtomc** [comment]
Thank you for finding this!

**Comment by femtomc** [comment]
Fixed on `nightly` @ 33031c35d6c6a165b7146bafc9feecc117e06b38.
